### PR TITLE
fix for RP-709

### PR DESCRIPTION
**Pull request by ftdholbrook**
- Jackson mixin for ResourceOwnerPasswordResourceDetails

### CODE REVIEW

File: admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/model/ApplicationTenantConfigurationPersistence.java [hunk 1]
```diff
@@ -84,10 +85,10 @@
...
     private Map<String, Map<String, Map<String, DataSourceElementsTenant>>> datasources;
     private Map<String, Map<String, ArchivePropertiesTenant>> archive;
-    private Map<String, Map<String, ExamProcessorValidationPropertiesTenant>> validation = ImmutableMap.of();
-    private Map<String, Map<String, ArtClientPropertiesTenant>> taskUpdateOrganizationsArtClient = ImmutableMap.of();
-    private Map<String, Map<String, ImportServiceClientPropertiesTenant>> taskUpdateOrganizationsImportServiceClient = ImmutableMap.of();
-    private Map<String, Map<String, SendReconciliationReportPropertiesTenant>> taskSendReconciliationReport = ImmutableMap.of();
+    private Map<String, Map<String, ExamProcessorValidationPropertiesTenant>> validation = new HashMap<>();
+    private Map<String, Map<String, ArtClientPropertiesTenant>> taskUpdateOrganizationsArtClient = new HashMap<>();
+    private Map<String, Map<String, ImportServiceClientPropertiesTenant>> taskUpdateOrganizationsImportServiceClient = new HashMap<>();
+    private Map<String, Map<String, SendReconciliationReportPropertiesTenant>> taskSendReconciliationReport = new HashMap<>();
```
Comment (ftdholbrook) [inline admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/model/ApplicationTenantConfigurationPersistence.java]: For reasons not clear to me, after adding the mixins in, the spring relaxed binding code started failing on these.  After trial and error replacing these with mutable hash maps fixed the problem.

File: admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/git/GitTenantConfigurationPersistenceService.java [hunk 2]
```diff
@@ -13,8 +13,10 @@
 import org.opentestsystem.rdw.admin.multitenant.model.ApplicationTenantConfigurationPersistence;
 import org.opentestsystem.rdw.admin.multitenant.model.TenantConfiguration;
 import org.opentestsystem.rdw.multitenant.Tenant;
+import org.opentestsystem.rdw.multitenant.task.ResourceOwnerPasswordResourceDetailsMixin;
```
Comment (gregcharles) [inline admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/git/GitTenantConfigurationPersistenceService.java]: Is this a new class? I can't find it. Does the common version need to be bumped? (ResourceOwnerPasswordResourceDetailsMixin)

Comment (ftdholbrook) [reply admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/git/GitTenantConfigurationPersistenceService.java]: https://github.com/SmarterApp/RDW_Common/pull/159

File: admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/model/ApplicationTenantConfigurationPersistence.java (hunk 1 above)
Comment (malaffoon) [reply admin-service/src/main/java/org/opentestsystem/rdw/admin/multitenant/model/ApplicationTenantConfigurationPersistence.java]: I was wondering about that during the initial checkin ... i've never been able to use immutable maps with spring property beans.

File: admin-service/src/test/java/org/opentestsystem/rdw/admin/multitenant/git/OjbectMapperTest.java [hunk 3]
```diff
@@ -46,10 +48,16 @@ public void bigTest() throws JsonProcessingException, BindException {
         final String output = ymlMapper.writeValueAsString(config);
         System.out.println(output);
```
Comment (malaffoon) [inline admin-service/src/test/java/org/opentestsystem/rdw/admin/multitenant/git/OjbectMapperTest.java]: should probably suppress this

**Pull request merged**